python/ray/tune/cluster_info.py

```diff
@@ -2,6 +2,10 @@
 import os
 
 
+def is_ray_cluster():
+    return os.path.exists(os.path.expanduser("~/ray_bootstrap_config.yaml"))
+
```

Comment:
Is this always true for ray clusters?

Reply:
yep - for all clusters started with the cluster launcher, left a comment